internal/gitprovider/github/github.go

```diff
@@ -146,13 +145,18 @@ func convertGithubPR(ghPR *github.PullRequest) *gitprovider.PullRequest {
 	}
 }
 
-func parseGitHubURL(u string) (string, string, error) {
-	regex := regexp.MustCompile(`^https\://github\.com/([\w-]+)/([\w-]+).*`)
-	parts := regex.FindStringSubmatch(u)
-	if len(parts) != 3 {
-		return "", "", fmt.Errorf("error parsing github repository URL %q", u)
+func parseGitHubURL(repoURL string) (string, string, error) {
+	u, err := url.Parse(repoURL)
+	if err != nil {
+		return "", "", fmt.Errorf("error parsing github repository URL %q: %w", u, err)
+	}
+	path := strings.TrimPrefix(u.Path, "/")
+	path = strings.TrimSuffix(path, ".git")
+	parts := strings.Split(path, "/")
+	if len(parts) != 2 {
+		return "", "", fmt.Errorf("could not parse github repository URL %q", u)
```

Comment:
```suggestion
		return "", "", fmt.Errorf("could not extract repository owner and name from URL %q", u)
```

maybe?

Reply:
Yes. I like this wording better. Thanks!